ocis-reva/CHANGELOG.md

```diff
@@ -0,0 +1,948 @@
+   created by the current user [(reva/#1042)](https://github.com/cs3org/reva/pull/1042)
+
+   https://github.com/owncloud/ocis/ocis-revapull/447
+
+# Changelog for [0.12.0] (2020-08-17)
+
+The following sections list the changes in ocis-reva 0.12.0.
+
+[0.12.0]: https://github.com/owncloud/ocis/ocis-revacompare/v0.11.0...v0.12.0
+
+## Summary
+
+* Bugfix - Update LDAP filters: [#399](https://github.com/owncloud/ocis/ocis-revapull/399)
+* Change - Environment updates for the username userid split: [#420](https://github.com/owncloud/ocis/ocis-revapull/420)
+* Enhancement - Update storage documentation: [#384](https://github.com/owncloud/ocis/ocis-revapull/384)
+* Enhancement - Update reva to v0.1.1-0.20200724135750-b46288b375d6: [#399](https://github.com/owncloud/ocis/ocis-revapull/399)
+* Enhancement - Update reva to v0.1.1-0.20200728071211-c948977dd3a0: [#407](https://github.com/owncloud/ocis/ocis-revapull/407)
+
+## Details
+
+* Bugfix - Update LDAP filters: [#399](https://github.com/owncloud/ocis/ocis-revapull/399)
+
+   With the separation of use and find filters we can now use a filter that taken into account a users
+   uuid as well as his username. This is necessary to make sharing work with the new account service
+   which assigns accounts an immutable account id that is different from the username.
+   Furthermore, the separate find filters now allows searching users by their displayname or
+   email as well.
+
+   ``` userfilter =
+   "(&(objectclass=posixAccount)(|(ownclouduuid={{.OpaqueId}})(cn={{.OpaqueId}})))"
+   findfilter =
+   "(&(objectclass=posixAccount)(|(cn={{query}}*)(displayname={{query}}*)(mail={{query}}*)))"
+   ```
+
+   https://github.com/owncloud/ocis/ocis-revapull/399
+   https://github.com/cs3org/reva/pull/996
+
+
+* Change - Environment updates for the username userid split: [#420](https://github.com/owncloud/ocis/ocis-revapull/420)
+
+   We updated the owncloud storage driver in reva to properly look up users by userid or username
+   using the userprovider instead of taking the path segment as is. This requires the user service
+   address as well as changing the default layout to the userid instead of the username. The latter
+   is not considered a stable and persistent identifier.
+
+   https://github.com/owncloud/ocis/ocis-revapull/420
+   https://github.com/cs3org/reva/pull/1033
+
+
+* Enhancement - Update storage documentation: [#384](https://github.com/owncloud/ocis/ocis-revapull/384)
+
+   We added details to the documentation about storage requirements known from ownCloud 10, the
+   local storage driver and the ownCloud storage driver.
+
+   https://github.com/owncloud/ocis/ocis-revapull/384
+   https://github.com/owncloud/ocis/ocis-revapull/390
+
+
+* Enhancement - Update reva to v0.1.1-0.20200724135750-b46288b375d6: [#399](https://github.com/owncloud/ocis/ocis-revapull/399)
+
+   - Update reva to v0.1.1-0.20200724135750-b46288b375d6 - Split LDAP user filters
```

Comment:
Codacy found an issue: [[unordered-list-marker-style] Marker style should be `*`](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)